api/client/formatter/custom.go

```diff
@@ -142,6 +143,20 @@ func (c *containerContext) Label(name string) string {
 	return c.c.Labels[name]
 }
 
+func (c *containerContext) Mounts() string {
```

Comment:
The function is exported because Go's template engine requires to export them, but the struct is not exported, so nobody else can really use it.